test/quantization/test_quantize.py

```diff
@@ -866,6 +866,49 @@ def test_default_quantized_lstm(self):
 
                 y, (h, c) = cell_dq(x, (h, c))
 
+    def test_per_channel_lstm_quantize(self):
+        d_hid = 2
+        num_chunks = 4
+        model = LSTMDynamicModel().eval()
+        cell = model.lstm
+        vals = [[100, -155],
+                [100, -155],
+                [-155, 100],
+                [-155, 100],
+                [100, -155],
+                [-155, 100],
+                [-155, 100],
+                [100, -155]]
+
+        vals = vals[:d_hid * num_chunks]
+        cell.weight_ih_l0 = torch.nn.Parameter(
+            torch.tensor(vals, dtype=torch.float),
+            requires_grad=False)
+        cell.weight_hh_l0 = torch.nn.Parameter(
+            torch.tensor(vals, dtype=torch.float),
+            requires_grad=False)
+        ref = copy.deepcopy(cell)
+        qconfig_dict = {
+            torch.nn.LSTM : per_channel_dynamic_qconfig
+        }
+        model_quantized = quantize_dynamic(model=model, qconfig_spec=qconfig_dict, dtype=torch.qint8)
+
+        niter = 10
+
+        x = torch.tensor([[100, -155],
+                          [-155, 100],
+                          [100, -155]], dtype=torch.float).unsqueeze(0).repeat(niter, 1, 1)
+
+        h0_vals = [[-155, 100],
+                   [-155, 155],
+                   [100, -155]]
+        hx = torch.tensor(h0_vals, dtype=torch.float).unsqueeze(0)
+        cx = torch.tensor(h0_vals, dtype=torch.float).unsqueeze(0)
+        hiddens = (hx, cx)
+        quant_out, quant_hidden = model_quantized(x)
```

Comment:
We need to check for scriptability and serialization, like in other tests. My PR should add this support for the baseline LSTM case, but hasnt landed yet.
```
            def checkQuantized(model):
                self.assertEqual(type(model.fc1), torch.nn.Linear)
                self.checkDynamicQuantizedLinear(model.fc2, dtype=dtype)
                self.checkScriptable(model, self.calib_data, check_save_load=True)
```